wavecrestweb/kenworthy-machine
# #KM-4/feature hero

## Description
This PR adds a hero section below the header with an image and a heading.


## Related Tickets & Documents
Closes #KM-4


## Acceptance Criteria

- [x] The hero section prominently displays a high-quality image that represents Kenworthy Machine's industry and services.
- [x] The title or heading within the hero section is clear and immediately informs the user of the company's core service offering.
- [x] The hero section design aligns with the company's branding and design aesthetics provided in the design files.
- [x] Text and imagery in the hero section are well-balanced and do not compete for attention, ensuring a clear message.
- [ ] The hero section is coded to accommodate various screen sizes without text or image distortion.




## What type of PR is this? (check all applicable)


|     | Type                       |
| --- | -------------------------- |
|   ✓  | :sparkles: New feature     |
|  ✓   | 🎨 Style                   |
|     | :hammer: Refactoring       |
|     | 🔥 Performance Improvements|
|     | :bug: Bug fix              |
|     | :100: Add tests            |
|     | :link: Update dependencies |
|     | :scroll: Docs              |


## Mobile & Desktop Screenshots/Recordings

### Before
<img width="1423" alt="Screenshot 2024-01-25 at 11 56 45 AM" src="https://github.com/wavecrestweb/kenworthy-machine/assets/115492619/15e99bde-32ba-46f9-9763-3c501bb3e8ee">

### After
<img width="717" alt="Screenshot 2024-01-25 at 11 50 15 AM" src="https://github.com/wavecrestweb/kenworthy-machine/assets/115492619/dbce3a2f-05b2-48eb-a7b7-60687b70a0cb">



## What I learned
I remembered the z-index prop for layering, which helped keep the sticky header on top. I also learned more about the properties for nextJS Image component.

## Testing steps

1. `git switch KM-4/feature-hero`
2. `npm run dev`
3. Open localhost:3000 and make sure the hero section is visible with the correct heading and image placements
4. Open the wireframes and check for any discrepancies


## Added to documentation?

|     | Type                       |
| --- | -------------------------- |
|     | 📜 README.md    |
|  ✓   | 🙅 no documentation needed    |


## What gif best describes this PR?
How I feel about no one adding gifs to their PRs:
  ![sad dog](https://media.giphy.com/media/v1.Y2lkPTc5MGI3NjExaGJqbWNyODBod3A1bXZpOGx6YnY4ajloZHdzamd5dmtoc3h0YXJsayZlcD12MV9pbnRlcm5hbF9naWZfYnlfaWQmY3Q9Zw/H4oLWS4veh9kqxQ1z2/giphy.gif)
